UPGRADE-6.0.md

```diff
@@ -173,6 +173,7 @@ Messenger
  * Removed the `prefetch_count` parameter in the AMQP bridge.
  * Removed the use of TLS option for Redis Bridge, use `rediss://127.0.0.1` instead of `redis://127.0.0.1?tls=1`
  * The `delete_after_ack` config option of the Redis transport now defaults to `true`
+ * The `reset_on_message` config option now defaults to `true`
```

Comment:
This line is in branch 5.4 but not in 6.0, both files should logically be the same.

Reply:
Oh probably because it's a FWB config and not related to the Messenger component? So it should be remove in 5.4 instead?

Reply:
see #43203

Reply:
It needs to be removed on 5.4 then.

Reply:
Done in https://github.com/symfony/symfony/pull/44496